01-guessing_game/randomGG.java

```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+public class Main
+{
+	public static void main(String[] args) {
+
+  Scanner in = new Scanner(System.in);
+    System.out.println("What is the minimum number you want to guess to?");
+    int min = in.nextInt();
+    System.out.println("What is the maximum number you want to guess to?");
+    int max = in.nextInt();
+
+		final int NUMBER = ((int)(Math.random() * max)+1);
+   String ans = null;
+   int guess = 0;
+   int guesses = 0;
+   String help = null;
+   boolean winGame = false;
+
+   while (winGame != true){//main loop //3
+     System.out.printf("What is your guess?(%d - %d) ",min,max);
+     if (guesses % 4 == 0 && guesses > 3){//4 //hint shows up every 4 loop
+       System.out.println("press any not number character to get hints");}
+
+     ans = in.next();//input string
+
+     boolean convertible = isInteger(ans);
+
+     if (convertible == false ){//4
+     System.out.println("type 'h' to know if the number is even or odd");
+     System.out.println("type 'f' to know first character of the random number");
+     System.out.println("type 'a' to give up and show answer");
+     help = in.next();
+     if (help.equals("h".toLowerCase())){//5 (lowercase dont work)
```

Comment:
i think you may want something like `"h".equals(help.toLowerCase())` so that the input is pushed to lower case.
this could be used in the other conditionals also.
OR
there's another String method, `equalsIgnoreCase(String anotherString)`
try it out!